Client/src/Components/Heading/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { Typography } from "@mui/material";
+import PropTypes from "prop-types";
+
+/**
+ * A heading component that renders text with a specific heading level.
+ *
+ * @param {Object} props - The properties passed to the Heading component.
+ * @param {('h1'|'h2'|'h3')} props.component - The heading level for the component.
+ * @param {string} props.children - The content to display inside the heading.
+ * @returns {JSX.Element} The Typography component with specified heading properties.
+ */
+
+function Heading({ component, children }) {
+	return (
+		<Typography
+			component={component}
+			variant="h2"
```

Comment:
Good point from coder rabbit, do we want to always use the `h2` styles here?

Reply:
Yeah, that one is midway done, I was going to implement it when doing the page, but got sidetracked by the theme. I'll work on it

Reply:
Oh, yeah, now I recall: the component prop defines the actual heading level, and the h2 is the variant. Right now all cases I saw where I could use this component were the variant h2. I will expand and add other variants as I find them on the code, ok?

Reply:
Yep the variant only applies styles, the component sets the actual HTML tag. 

Sounds good, we can update as required :+1: